librz/core/cmd_descs/cmd_debug.yaml

```diff
@@ -1009,3 +1009,36 @@ commands:
             args:
               - name: number
                 type: RZ_CMD_ARG_TYPE_NUM
+  - name: dx
```

Comment:
I'd be a bit more descriptive on what these commands do because they are quite nice feature and they are not well explained at the moment. Maybe even add a `description` if necessary.

Reply:
I added examples in `details` section. Hopefully, that is enough.